spring-cloud-azure-starters/spring-cloud-starter-azure-appconfiguration-config/README.md

```diff
@@ -37,77 +40,126 @@ spring.cloud.azure.appconfiguration.fail-fast | Whether throw RuntimeException o
 spring.cloud.azure.appconfiguration.stores[0].prefix=[my-prefix]
 spring.cloud.azure.appconfiguration.stores[0].label=[my-label]
 spring.cloud.azure.appconfiguration.stores[1].connection-string=[second-store-connection-string]
 ```
+
 If duplicate keys exists for multiple stores, the last configuration store has the highest priority.
 
 ### Load from multiple labels
+
 If the application needs to load property values from multiple labels in the same configuration store, following configuration can be used:
-```
+
+```properties
 spring.cloud.azure.appconfiguration.stores[0].connection-string=[first-store-connection-string]
 spring.cloud.azure.appconfiguration.stores[0].label=[my-label1], [my-label2]
 ```
+
 Multiple labels can be separated with comma, if duplicate keys exists for multiple labels, the last label has highest priority.
 
 ### Watch configuration change
+
 Watch feature allows the application to load the latest property value from configuration store automatically, without restarting the application.
 
 By default, the watch feature is disabled. It can be enabled with below configuration:
-```
+
+```properties
 spring.cloud.azure.appconfiguration.watch.enabled=true
 ```
 
 Change certain property key in the configuration store on Azure Portal, e.g., /application/config.message, log similar with below will be printed on the console.
-```
+
+```console
 INFO 17496 --- [TaskScheduler-1] o.s.c.e.event.RefreshEventListener       : Refresh keys changed: [config.message]
 ```
+
 The application now will be using the updated properties. By default, `@ConfigurationProperties` annotated beans will be automatically refreshed. Use `@RefreshScope` on beans which are required to be refreshed when properties are changed.
 By default, all the keys in a configuration store will be watched. To prevent configuration changes are picked up in the middle of an update of multiple keys, you are recommended to use the watched-key property to watch a specific key that signals the completion of your update so all configuration changes can be refreshed together.
-```
+
+```properties
 spring.cloud.azure.appconfiguration.stores[0].watched-key=[my-watched-key]
 ```
 
 ### Failfast
+
 Failfast feature decides whether throw RuntimeException or not when exception happens. By default, failfast is enabled, it can be disabled with below configuration:
-```
+
+```properties
 spring.cloud.azure.appconfiguration.fail-fast=false
 ```
 
-### Use Azure Managed Identity to load the connection string
-[Managed service identity](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview) allows application to access [Azure Active Directory](https://azure.microsoft.com/en-us/services/active-directory/) protected resource on [Azure](https://azure.microsoft.com/en-us/).
+### Use Azure Managed Identity/Service Principle to load the connection string
```

Comment:
I find combining managed identity and service principle is very confusing. We should separate them. For example, you can have two sections
#### Use Managed Identity to access App Configuration
#### Use Azure Active Directory to access App Configuration

There is nothing about loading connection string here. It should be removed.